### PR TITLE
Add Fast DDS-Gen subsection to Topic documentation [8569]

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/451/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/452/)

File: docs/fastdds/dds_layer/topic/typeSupport/includes/fastddsgen.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.. _dds_layer_topic_fastddsgen:
+
+Fast DDS-Gen for data types source code generation
+--------------------------------------------------
+
+*eProsima Fast DDS* comes with a built-in source code generation tool, Fast DDS-Gen, which eases the process of
+translating an IDL specification of a data type to a functional implementation.
+Thus, this tool automatically generates the source code of a data type defined using IDL.
+A basic use of the tool is described below.
+To learn about all the features that Fast offers, please refer to :ref:`Fast DDS-Gen <fastrtpsgen_intro>` section .
```
Comment (EduPonz) [inline docs/fastdds/dds_layer/topic/typeSupport/includes/fastddsgen.rst:10]: Incomplete `that Fast offers`

File: docs/fastdds/dds_layer/topic/typeSupport/includes/fastddsgen.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.. _dds_layer_topic_fastddsgen:
+
+Fast DDS-Gen for data types source code generation
```
Comment (EduPonz) [inline docs/fastdds/dds_layer/topic/typeSupport/includes/fastddsgen.rst:3]: I think the section is to hidden. I'd actually put it as a children of Topic section

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/455/)

**Pull request merged**